### ISSUE TITLE
Can not connect to matrix server because of the unnecessary slash

### ISSUE THREAD

**Issue by Lily-Ayta** [QMatrixClient/Quaternion#222]
In Quaternion 0.0.4 (compiled with Qt 5.9.3, GCC 7.2.0)

```
sh-4.4$ quaternion
qt5ct: using qt5ct plugin
Gtk-Message: Failed to load module "canberra-gtk-module"
Debug:  false
qml: onCompleted
qt5ct: D-Bus global menu: no
qt5ct: D-Bus system tray: no
libqmatrixclient.jobs: "LoginJob" status 104 : "Error transferring https://matrix.org//_matrix/client/r0/login - server replied: Not Found"
--- Show time!
QXcbConnection: XCB error: 3 (BadWindow), sequence: 1136, resource id: 11019541, major code: 40 (TranslateCoords), minor code: 0
Last window closed!
sh-4.4$ curl https://matrix.org//_matrix/client/r0/login
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /_matrix/client/r0/login was not found on this server.</p>
<hr>
<address>Apache/2.4.10 (Debian) Server at matrix.org Port 443</address>
</body></html>
sh-4.4$ curl https://matrix.org/_matrix/client/r0/login
{
    "flows": [
        {
            "type": "m.login.password"
        }
    ]
}
sh-4.4$ 
```

![default](https://user-images.githubusercontent.com/29628836/33992578-2715c552-e10e-11e7-8c01-8a514ab2c69f.png)

**Comment by Lily-Ayta** [comment]
Confirmed this problem appeared for the older libqmatrixclient used. It may be fixed in [libqmatrixclient @ fed4b4f](https://github.com/QMatrixClient/libqmatrixclient/commit/fed4b4fe965a9e2788055b602c7217cb298f5fce) 

Ref: [libqmatrixclient @ #125](https://github.com/QMatrixClient/libqmatrixclient/issues/125)

**Comment by KitsuneRal** [comment]
Thank you for reporting. I assume you use Quaternion with Qt 5.10; they have changed the way they process URLs. And yes, this is covered by #125.
